### PR TITLE
important pr oranges asked me to make

### PR DESCRIPTION
**Pull request by scriptis**
## About The Pull Request

adds an important feature that oranges okayed during the headmin debate despite the feature freeze
## Why It's Good For The Game

orange man says

image

this is a joke pr
## Changelog
:cl:
oversight: copies some important qol features that were only working for aliens to felinids
/:cl:

### CODE REVIEW

**Comment by arnociiroc** [comment]
speedmerged

**Comment by TJatPBnJ** [comment]
> this is a joke pr

can you remove this part of the pr body

**Comment by scriptis** [comment]
no

File: code/modules/mob/living/carbon/alien/adult/alien_powers.dm
Comment (scriptis) [reply code/modules/mob/living/carbon/alien/adult/alien_powers.dm:401]: works on my machine

Comment (scriptis) [reply code/modules/mob/living/carbon/alien/adult/alien_powers.dm:401]: watch this sick maneuver

**Comment by Fritzlschnitzel** [comment]
looks good.

**Comment by RigglePrime** [comment]
Science has gone too far

**Comment by iwishforducks** [comment]
Code looks good 😋

**Comment by Wallemations** [comment]
Thank you for citing your sources

**Review by ZephyrTFA — APPROVED** [review]

**Comment by epicgamer545** [comment]
I'm leaving the TG community

**Comment by scriptis** [comment]
WHO RAN THIS ON MANUEL LMAO

**Comment by private-tristan** [comment]
Speede mergeh

**Comment by SamDDBS** [comment]
i think this needs to be atomized first

**Comment by Kubisopplay** [comment]
Which madman ran it on live servers holy fuck

**Comment by arnociiroc** [comment]
perma merged???

**Comment by the-orange-cow** [comment]
Based?  Based on what?

**Comment by TheVekter** [comment]
The V word

**Comment by TheVekter** [comment]
> Based? Based on what?

Based on the hardware that's installed on it.

**Comment by iwishforducks** [comment]
> The V word

based on Vekter

**Review by Kylerace — APPROVED** [review]

**Review by optimumtact — APPROVED** [review]
approved pending other headcoders approval

**Comment by Archie700** [comment]
>felinids can't strip
literal nerf, wtf, now I can't do my "catgirl dorms 4" antag gimmick

**Comment by Mothblocks** [comment]
> approved pending other headcoders approval

U didn't message me!!!!!!!!

**Pull request closed without merging**
